### ISSUE TITLE
MP extension openapi-ui does not seem to work with Helidon

### ISSUE THREAD

**Issue by tjquinno** [oracle/helidon#1385]
## Environment Details
* Helidon Version: 1.4.x and 2.0.0-M1
* Helidon SE or Helidon MP: MP
* JDK version: 11
* OS: n/a
* Docker version (if applicable): n/a

----------

## Problem Description
(This was originally reported by a user who is currently using the Firefox plug-in as a work-around.)

After adding the openapi-ui MP extension (https://github.com/microprofile-extensions/openapi-ext) to a Helidon app, accessing the endpoint `/openapi-ui` renders only the MP logo image; it does not show the expected U/I that would allow the user to exercise the app via the generic OpenAPI-driven page.

## Steps to reproduce
* Add the extension to the pom: 
```
<dependency>
    <groupId>org.microprofile-ext.openapi-ext</groupId>
    <artifactId>openapi-ui</artifactId>
    <version>1.1.0</version>
    <scope>runtime</scope>
</dependency>
```
* Make the OpenAPI U/I services discoverable:
    * Using Helidon 1.4, add these two OpenAPI U/I service classes to the set returned by the application's `getClasses` method: `OpenApiUiService`, `StaticResourcesService`.
    * Using Helidon 2.x, either add the services as above or comment out the `getClasses` method entirely and let Helidon discover all the services automatically.
* Build and run the app.
* Access the endpoint `/openapi-ui` (which redirects to that path plus `index.html`).

Using a browser (at least Safari and Firefox) shows only the MP logo image; no U/I to drive the app's API is rendered.

Using `curl` shows this output:
```
HTTP/1.1 200 OK
Content-Type: text/html
Date: Wed, 12 Feb 2020 16:59:23 -0600
connection: keep-alive
content-length: 2000

<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>MicroProfile - Open API</title>
        <link rel="icon" type="image/png" href="favicon-32.png" sizes="32x32" />
        <link rel="icon" type="image/png" href="favicon-16.png" sizes="16x16" />

        <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Source+Code+Pro:300,600|Titillium+Web:400,600,700" rel="stylesheet" type="text/css"/>
        
        <link href="//webjars/swagger-ui/3.24.3/swagger-ui.css" rel="stylesheet" type="text/css" />
        <link href="//webjars/swagger-ui-themes/3.0.0/themes/3.x/theme-flattop.css" rel="stylesheet" type="text/css" />
        <link href="style.css" rel="stylesheet" type="text/css" />
    </head>

    <body>
        <div id="swagger-ui"></div>
        <script src='//webjars/swagger-ui/3.24.3/swagger-ui-bundle.js' type='text/javascript'></script>
        <script src='//webjars/swagger-ui/3.24.3/swagger-ui-standalone-preset.js' type='text/javascript'></script>
        <script>
            window.onload = function () {

                // Build a system
                const ui = SwaggerUIBundle({
                    url: "/openapi",
                    dom_id: '#swagger-ui',
                    deepLinking: true,
                    presets: [
                        SwaggerUIBundle.presets.apis,
                        SwaggerUIStandalonePreset
                    ],
                    plugins: [
                        SwaggerUIBundle.plugins.DownloadUrl
                    ],
                    layout: "StandaloneLayout"
                })

                window.ui = ui
            }
        </script>
        
        <div id='footer'>
            <p class="swagger-ui-wrap">
                <span class="createdWith">Created with <a target="_blank" href="http://www.microprofile-ext.org" data-sw-translate>MicroProfile Extensions: OpenApi</a></span> . &copy; 2020 
            </p>
        </div>
    </body>
</html>
```
Using Safari's developer mode to view the page source, the resources from `/webjars` -- `swagger-ui-bundle.js`, `swagger-ui.css`, and `there-flattop.css` -- show as `Resource has no content` so that seems to be at least part of the problem.

**Comment by tjquinno** [comment]
After further investigation I've concluded that this is a problem in the extension.

The extension contains logic which computes the context root from the base URI and adds a `"/"` prefix to it. For Helidon, though, the base URI is `http://[0:0:0:0:0:0:0:1]:8080/` which leads to a context root of `"/"`. When this is substituted into their templates, it leads to URIs of the form `//webjars...` which fail.

If I use the debugger to force the context root to be empty, then things work.

I've added to the existing issue Joe had opened (since they had asked there for us to check to see if their latest changes worked for us) https://github.com/microprofile-extensions/openapi-ext/issues/15 with what I found and I attached a reproducer.

I'll leave this open so we track progress on the extension issue.
